ext/hal/nordic/nrfx_glue.h

```diff
@@ -176,6 +176,37 @@ extern "C" {
  */
 #define NRFX_TIMERS_USED        0
 
+//------------------------------------------------------------------------------
+
+/**
+ * @brief Macro for getting the interrupt number assigned to a specific
+ *        peripheral.
+ *
+ * In Nordic SoCs the IRQ number assigned to a peripheral is equal to the ID
+ * of this peripheral, and there is a direct relationship between this ID and
+ * the peripheral base address, i.e. the address of a fixed block of 0x1000
+ * bytes of address space assigned to this peripheral.
+ * See the chapter "Peripheral interface" (sections "Peripheral ID" and
+ * "Interrupts") in the product specification of a given SoC.
+ *
+ * @param[in] base_addr  Peripheral base address.
+ *
+ * @return Interrupt number associated with the specified peripheral.
+ */
+/* TODO - this macro should become a part of <nrfx_common.h> */
```

Comment:
Can you add an entry in the nrfx bug tracking system to get rid of this once it's part of nrfx_common.h ?

Reply:
[Done](https://github.com/NordicSemiconductor/nrfx/issues/18).